samples/Test.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Since this is XML, and it has lint in it, could we (and this is a bit meta, I realise) find and add an XML linter to this repo's .pre-commit-config.yaml?

Reply:
[This repo](https://github.com/pre-commit/pre-commit-hooks) has a check-xml hook, but would require updating the pinned version. If there are no issue with me doing that, happy to bump it up to 3.2.0.